doc/ipython-notebooks/clustering/GMM.ipynb

```diff
@@ -318,17 +307,14 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "def estimate_gmm(X, num_components):\n",
     "    # bring data into shogun representation (note that Shogun data is in column vector form, so transpose)\n",
-    "    features=RealFeatures(X.T)\n",
     "    \n",
     "    gmm_est=GMM(num_components)\n",
-    "    gmm_est.set_features(features)\n",
+    "    gmm_est.set_features(features(X.T))\n",
```

Comment:
can you use put here?

Reply:
actually we can't, GMM is a  CDistrubition, where 'features' is not registered

Reply:
ok!